### PR TITLE
added a value attribute to the attack pattern ioc

### PR DESCRIPTION
**Pull request by omerKarkKatz**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/47980)

## Description
added a value attribute to the context of the attack pattern reputation command  

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by omerKarkKatz** [comment]
@yaakovi didnt update the docker image because it is breaking the backwards computability and this issue is a blocker for other issues, and needs to be merged ASAP.

File: Packs/Base/Scripts/CommonServerPython/CommonServerPython.py [hunk 1]
```diff
@@ -4840,7 +4840,7 @@ class AttackPattern(Indicator):
 
         def __init__(self, stix_id, kill_chain_phases=None, first_seen_by_source=None, description=None,
                      operating_system_refs=None, publications=None, mitre_id=None, tags=None,
-                     traffic_light_protocol=None, dbot_score=None, community_notes=None, external_references=None):
+                     traffic_light_protocol=None, dbot_score=None, community_notes=None, external_references=None, value=None):
```
Comment (dorschw) [inline Packs/Base/Scripts/CommonServerPython/CommonServerPython.py:4843]: please add an example for `value` in a docstring, as its name is ambiguous

File: Packs/Base/ReleaseNotes/1_30_9.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+- **CommonServerPython**
```
Comment (yaakovi) [inline Packs/Base/ReleaseNotes/1_30_9.md:3]:
```suggestion
##### CommonServerPython
```

File: Packs/FeedMitreAttackv2/ReleaseNotes/1_1_1.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+- **MITRE ATT&CK Feed v2**
+- Improved the human-readable in the ***attack_pattern_reputation_command***.
+- Added a *AttackPattern.Value* to the context entry of the ***attack_pattern_reputation_command***.
```
Comment (yaakovi) [inline Packs/FeedMitreAttackv2/ReleaseNotes/1_1_1.md:5]:
```suggestion
##### MITRE ATT&CK Feed v2
- Improved the human-readable in the ***attack-pattern*** command.
- Added the *AttackPattern.Value* output to the ***attack-pattern*** command.
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**